packages/bootstrap/scss/pivotgrid/_layout.scss

```diff
@@ -1 +1,9 @@
 @import "~@progress/kendo-theme-default/scss/pivotgrid/_layout.scss";
+
+@include exports ("pivotgrid/layout/bootstrap/legacy") {
+
+    .k-pivot-toolbar {
+        padding: $toolbar-padding-y;
```

Comment:
if we want to override a variable, we create a local one. In this case, two local ones: `$pivotgrid-toolbar-padding-x` and `$pivotgrid-toolbar-padding-y` or we don't customize the padding at all

Reply:
Vanka, this line of code is inside the legacy styles of the Pivotgrid. I didn't touch the styles there, only place them under `/legacy` module. In the current rendering, there is no toolbar at all.

Reply:
I noticed that in `https://github.com/telerik/kendo-themes/pull/2606`, you've removed these pivot toolbar styles. It might be my mistake when I rebased. I'll erase them.